### PR TITLE
website: Add "Exemptions" section to Contributing

### PR DESCRIPTION
**Pull request by BobbyRBruce**
This section explains that the stable branch may be updated if files within the ".github" directory are being updated.

### CODE REVIEW

**Comment by BobbyRBruce** [comment]
Sorry for this additional addition @powerjg . I forgot to add it in the original update but feel it should be written down somewhere.

File: _pages/contributing.md.bak
Comment (powerjg) [inline _pages/contributing.md.bak:1]: Did you mean to include this file?

**Review by powerjg — COMMENTED** [review]
Other than the .bak file, this is good with me.

File: _pages/contributing.md.bak
Comment (BobbyRBruce) [reply _pages/contributing.md.bak:1]: No. It's removed.

**Review by powerjg — APPROVED** [review]

**Pull request merged**